tests/test_awaitable_task.py

```diff
@@ -103,7 +103,7 @@ def test_task_monitor(decorated_add):
     #
     wg1 = WorkGraph(name="wg1")
     wg1.add_task(decorated_add, "add1", x=1, y=2, t=5)
-    wg1.submit(wait=True)
+    wg1.submit(wait=True, timeout=60)
```

Comment:
Do we raise an error if we surpass the timeout? I am just worried that the test passes due to the timeout even though it should not

Reply:
I just checked, and it will raise an error when it surpasses the timeout.